docker/development/docker-compose.yml

```
@@ -0,0 +1,13 @@
+version: "3.8"
+
+include:
+  - supabase/docker/docker-compose.yml
+
+services:
+  supabase:
+    build:
+      context: ../../
+      dockerfile: ./Dockerfile
+    depends_on:
+      - kong # defined by another_domain
+    
```

To run this locally (I presume this is the usecase we're covering with this `development` directory), I needed to add the following:
```suggestion
    ports:
      - "0.0.0.0:3000:3000"
```

---

3000:3000 maps the port exposed by the dockerfile to the localhost, this works